playbooks/code-review.md

```diff
@@ -2,3 +2,45 @@
 title: Code reviews
 description: How our code review process works 
 ---
+
+Reviewing other people's code is a responsibility of every single developer. Instead of viewing it as a hassle, it should be looked upon as a necessary process in order to keep the code clean and consistent.
+It is also an opportunity to learn something new.
+
+Rules on..
+
+## How a Code Review process should look like
+
+..are written below.
+
+### Terms:
+ - PR - Pull Request.
+ - PR Creator - a person who initiated the PR.
+ - Reviewer - a person assigned to do a PR code review.
+
+### Rules:
+1. Every PR has to be reviewed, with the exception of fix/hotfix PR-s that can be merged by the code owner on his own responsibility.
+2. When creating a PR, you should select **EXACTLY ONE** person as a reviewer. This person has to be someone who is working at the moment (not on vacation). 
+If there is someone else who should see this PR, he/she should be assigned as Assignee on this PR.
+	Reviewer Priority list:
+	1. Most senior developer on the project (excluding you).
```

Comment:
Code owner should be the first person that comes to mind, no?
We could use "official" CODEOWNERS syntax files - https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

Reply:
Do we have code owners in our organization? I agree, code owner should be the first on the list.

Reply:
Codeowners is just a simple file, .gitignore style, it can be per project, per folder, or even seems to work per file. So it's not an organization thing. Just add it to your favorite project and you're set :)

Reply:
Added CODEOWNERS to the list